### PR TITLE
Add metadata to mask output

### PR DESCRIPTION
**Pull request by joshmoore**
* [x] register list of masks (simple)
 * [x] formalize list of masks
 * [ ] add groups for subresolutions --> https://github.com/ome/omero-cli-zarr/issues/17
 * [ ] formalize colors --> https://github.com/ome/omero-cli-zarr/issues/16

### CODE REVIEW

**Comment by joshmoore** [comment]
@will-moore : current "lookup" for masks is the super simple:

```
(demo) /opt/omero-cli-zarr $cat 6001240.zarr/masks/.zattrs
{
    "masks": [
        "0"
    ]
```

**Comment by jburel** [comment]
``masks = requests.get(f'{endpoint}/{imagepath}/masks/.zattrs').json()``
returns
``{'masks': ['0']}``

**Comment by jburel** [comment]
Screenshot 2020-07-03 at 15 53 43

**Comment by jburel** [comment]
I can see putting the probabilities map coming for example from ilastik, going under masks. Info about that mask (tool, version, maybe info about the setup itself i.e. parameters used) will need to be added.

**Comment by jburel** [comment]
I started to look at analysing http://idr.openmicroscopy.org/webclient/img_detail/4007801/
This image has a large number of timepoints and z-sections: many are black and not very useful.
I regenerate a "segmented" 5D image after analysing the plane in parallel. This image has the same dimension that the original one.
I am wondering if we should not allow for "smaller" image and indicate the range used in the json e.g. zstart/end etc.
but that could become too complex. The downside is that we will use more storage than needed.

**Comment by jburel** [comment]
Screenshot 2020-07-08 at 15 11 59
Comparing image and mask in 3D using ITK

**Comment by jburel** [comment]
Screenshot 2020-07-08 at 15 18 24
Combine the view in itk

File: src/omero_zarr/cli.py [hunk 1]
```diff
@@ -123,7 +126,9 @@ def _configure(self, parser):
             "--style",
             choices=("6d", "split", "labelled"),
             default="labelled",
-            help=("Choice of storage for ROIs"),
+            help=(
+                "Choice of storage for ROIs [breaks ome-zarr]"
```
Comment (will-moore) [inline src/omero_zarr/cli.py:130]: "Split" doesn't break ome-zarr (since it reads names of masks)? - I'll test to check...

Comment (will-moore) [reply src/omero_zarr/cli.py:130]: Yes - works for me (once I'd remembered to checkout the right branch!)

Screenshot 2020-07-09 at 10 04 23

**Comment by joshmoore** [comment]
> Info about that mask (tool, version, maybe info about the setup itself i.e. parameters used)...

see https://github.com/ome/omero-cli-zarr/issues/18

File: src/omero_zarr/cli.py (hunk 1 above)
Comment (joshmoore) [reply src/omero_zarr/cli.py:130]: Were the splits registered in `masks/.zattrs`? If not, that would need adding.

Comment (will-moore) [reply src/omero_zarr/cli.py]: yes - that's what `ome-zarr` was using to load them into napari. This shows the 'split' masks as well as the 'labelled 0` mask:

```
$ cat 52.zarr/masks/.zattrs
{
    "masks": [
        "208516",
        "208517",
        "208518",
        "208519",
        "208520",
        "208525",
        "208526",
        "208527",
        "208528",
        "0"
    ]
```

**Comment by will-moore** [comment]
This is good to merge from me.
Since the unchecked points in the description are moved to other issues, and this PR is required for the current state of `ome-zarr-py` to work, I think this can be merged now.
@jburel ?

**Comment by sbesson** [comment]
Thanks all. Merging and the next steps and todos can be captured as separate issues

**Pull request merged**